extensions/amp-story/1.0/amp-story-desktop-panels.css

```diff
@@ -180,7 +180,8 @@ amp-story[standalone].i-amphtml-story-desktop-panels {
 }
 
 .i-amphtml-story-desktop-panels > amp-story-page,
-.i-amphtml-story-desktop-panels .i-amphtml-story-page-sentinel {
+.i-amphtml-story-desktop-panels .i-amphtml-story-page-sentinel,
+.i-amphtml-story-desktop-panels .i-amphtml-story-media-query-matcher {
```

Comment:
Oh woops, didn't see this. Disregard my other comment re. size of iframe.

Reply:
Hmm, now that I'm thinking about it, I wonder if this is fragile?  We will need to always remember with new UI types that we add to update this as well.  Since embedding in the active page is also tricky since the active page changes, maybe we can consider using the CSS vars `--i-amphtml-story-vh` and `--i-amphtml-story-vw` to just size the iframe at `100vw` and `100vh`?

Reply:
It is fragile, we'll have to maintain it and update it as we add new UI types. However, I believe it makes the UX so much better that it is worth the risk.
The CSS variables are updated on `resize` events, which are debounced. At best, the media queries would trigger after 300ms.
With the current implementation, they trigger instantly on resize.

For example, if you have one portrait UI and one landscape UI, using the CSS variables you'd see the portrait UI for at least 300ms when rotating your phone to landscape.
But with the current implementation, the UI would adapt right away.
It's a better experience, and it'll allow us to perform visibility checks on media elements without being worried of race conditions.

Reply:
Good call.  Do you mind filing a cover bug for adding contributor-facing documentation on UI types, and mentioning this in that bug?  We can't always ensure everything in code, and maybe we should rely more on actually having documentation for contributors 😅

Reply:
Sg, I filed https://github.com/ampproject/amphtml/issues/21981